Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -5062,7 +5062,11 @@ static BOOL shouldUseHighlightsForMarkedText(NSAttributedString *string)
         if (underline.thick)
             mergedUnderlines.append(underline);
     }
-#else
+
+    if (mergedUnderlines.size())
```

Comment:
should probably be !mergedUnderlines.isEmpty()

Reply:
Just wondering why would you prefer that when the implementation of isEmpty is `return !size()` ?